lib/text.rb

```
@@ -0,0 +1,32 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Text
+  TIME_FORMAT = "%k:%M"
+
+  attr_reader :mobile
+  
+  def initialize
+    @mobile = ENV['CUST_NUMBER']
+  end
+  
+  def send_message
+    account_sid = ENV['ACCOUNT_SID']
+    auth_token = ENV['AUTH_TOKEN']
+    client = Twilio::REST::Client.new(account_sid, auth_token)  
+    
+    begin
+      client.messages.create(
+        body: "Thank you! Your order was placed and will be delivered before #{delivery_time}",
+        to: @mobile,
+        from: ENV['TWILIO_NUMBER']
+      )
+      puts message.sid
+    rescue
```

This empty rescue block just swallows any errors, without telling you / the user about them.

I'd encourage you to look into whether you can rescue a more specific error, and at least `puts` to share that an error occurred.